qiskit_experiments/library/characterization/half_angle.py

```diff
@@ -0,0 +1,139 @@
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Half angle characterization."""
+
+from typing import List, Optional
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.providers import Backend
+
+from qiskit_experiments.framework import BaseExperiment, Options
+from qiskit_experiments.library.calibration.analysis import FineHalfAngleAnalysis
+from qiskit_experiments.curve_analysis import ParameterRepr
+
+
+class HalfAngle(BaseExperiment):
+    r"""An experiment class to measure the amount by which sx and x are not parallel.
+
+    # section: overview
+
+        This experiment runs circuits that repeat blocks of :code:`sx - sx - y` gates
+        inserted in a Ramsey type experiment, i.e. the full gate sequence is thus
+        :code:`Ry(π/2) - [sx - sx - y] ^ n - sx` where :code:`n` is varied.
+
+        .. parsed-literal::
+
+                    ┌─────────┐┌────┐┌────┐┌───┐   ┌────┐┌────┐┌───┐┌────┐ ░ ┌─┐
+               q_0: ┤ Ry(π/2) ├┤ sx ├┤ sx ├┤ y ├...┤ sx ├┤ sx ├┤ y ├┤ sx ├─░─┤M├
+                    └─────────┘└────┘└────┘└───┘   └────┘└────┘└───┘└────┘ ░ └╥┘
+            meas: 1/════════════════════════════...═══════════════════════════╩═
+                                                                              0
+
+        This sequence corrects angle errors where the axis of the :code:`sx` and :code:`x`
+        rotation are not parallel. A similar experiment is described in Ref.~[1] where the
+        gate sequence :code:`x - y` is repeated to amplify errors caused by non-orthogonal
+        :code:`x` and :code:`y` rotation axis. Such errors can occur due to phase errors.
+        For example, the non-linearities in the mixer's skew for :math:`\pi/2` pulses may
+        be different from the :math:`\pi` pulse.
+
+    # section: reference
+        .. ref_arxiv:: 1 1504.06597
+    """
+
+    __analysis_class__ = FineHalfAngleAnalysis
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        r"""Default values for the half angle experiment.
+
+        Experiment Options:
+            repetitions (List[int]): A list of the number of times that the gate
+                sequence :code:`[sx sx y]` is repeated.
+        """
+        options = super()._default_experiment_options()
+        options.repetitions = list(range(15))
+        return options
+
+    @classmethod
+    def _default_transpile_options(cls) -> Options:
+        """Default transpile options."""
+        options = super()._default_transpile_options()
+        options.basis_gates = ["sx", "rz", "y"]  # Should not be changed.
```

Comment:
Does "should not be changed" need to be in a doc string so the user sees it?

Reply:
Done here: 756251a